lib/contact_fixer.rb

```diff
@@ -2,10 +2,19 @@
 
 class ContactFixer
   def initialize(contacts_api, output)
```

Comment:
why not allow the filter to be set here aswell?

Reply:
Done